src/index.js

```diff
@@ -39,6 +39,6 @@ el10.addEventListener("click", function() {
 	// 	console.log("logkey", log[key]);
 	// });
 
-    root.innerHTML = `<pre><code>${JSON.stringify(world, undefined, 4)}</code></pre>`;
-	log.innerHTML = `<pre>${JSON.stringify(window.logger.getLog(), undefined, 4)}</pre>`;
+    // root.innerHTML = `<pre><code>${JSON.stringify(world, undefined, 4)}</code></pre>`;
```

Comment:
`JSON.stringify`ing large objects/arrays can take a while and so can rendering them. This was.

![image](https://user-images.githubusercontent.com/601509/29007505-d6570c3c-7ac1-11e7-8366-cb02b5f8a8e6.png)


The purple and the blue are from the call to `JSON.stringify` and layout updates. Commenting these two lines out completely removes those bars:

![image](https://user-images.githubusercontent.com/601509/29007529-3057eac6-7ac2-11e7-95ce-620ff5d49f69.png)